lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -8,7 +8,7 @@
     %>
     <div class='exam-text js-exam-text' data-show-long="true">
         <% // xss-lint: disable=underscore-not-escaped %>
-        <%= interpolate_text('You are taking "{exam_link}" as a {exam_type} exam. ', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
+        <%= interpolate_text('You are taking "{exam_link}" as {exam_type}. ', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
```

Comment:
I'm skeptical that this way of creating the string will be correctly translated in all languages. 

Also, shouldn't it be `gettext('a timed exam')`?

Reply:
I doubt it was well set up to be translated in all languages before